### PR TITLE
Fix build error of lilac, as it still uses the version from aur

### PR DESCRIPTION
**Pull request by ideal**
Or else the build would fail.

### CODE REVIEW

**Comment by ideal** [comment]
@yan12125 After the last merge, lilac build still fail, because lilac.py is not changed and it still uses aur files.

**Comment by yan12125** [comment]
Some major issues:

1. You need to change [the [systemtap] section](https://github.com/archlinuxcn/repo/blob/12c0fa3f7ea67ae1dabdb7ea5f1123b441214844/nvchecker.ini#L1186) from nvchecker.ini if you want to keep this package independent from the AUR one.
2. You changed the maintainer from @lijinpei to yourself. Have you got the permission?

**Comment by ideal** [comment]
No, so need to wait @lijinpei .

**Comment by yan12125** [comment]
Let me make a dictatorial decision here. I consider the new `maintainer` section in lilac.yaml of this PR an orphan request, and I want to follow the AUR convention to accept this request after 14 days without response from the original maintainer. The maintainer of the `systemtap` package is now @ideal.

@ideal, please go on for remaining parts. You may want to rebase and fix merge conflicts caused by [lilac.yaml migration](https://github.com/archlinuxcn/repo/issues/923) first.

**Comment by ideal** [comment]
OK.

**Comment by ideal** [comment]
Have rebased from archlinuxcn/master.

**Comment by petronny** [comment]
。。。如果还没回复你就去发一个orphan request

File: systemtap/lilac.yaml [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
 
-build_prefix: archlinuxcn-x86_64
-
-pre_build: aur_pre_build
-
-post_build: aur_post_build
+build_prefix: extra-x86_64
 
 update_on:
   - aur: systemtap
```
Comment (yan12125) [inline systemtap/lilac.yaml]: This line should be changed. With the current configuration, the package will be rebuilt when the AUR package is updated. In this case the same PKGBUILD is processed again and thus wasting server resources.

Comment (ideal) [reply systemtap/lilac.yaml]:
```
diff --git a/systemtap/lilac.yaml b/systemtap/lilac.yaml
index d90102f31..b0f5b6fd1 100644
--- a/systemtap/lilac.yaml
+++ b/systemtap/lilac.yaml
@@ -7,4 +7,4 @@ maintainers:
 build_prefix: extra-x86_64

 update_on:
-  - aur: systemtap
+  - aur
```

Will this be OK?

Comment (yan12125) [reply systemtap/lilac.yaml]: Well,  `- aur` is equivalent to `- aur: systemtap` - lilac determines the AUR package name if it's omitted. You may want to use `manual` in this case.

**Comment by yan12125** [comment]
Done! Thanks for the fix!

**Pull request merged**